src/Compilers/CSharp/Portable/SymbolDisplay/SymbolDisplay.cs

```diff
@@ -67,15 +75,25 @@ public static class SymbolDisplay
 #pragma warning disable RS0026 // Do not add multiple public overloads with optional parameters
         public static string ToMinimalDisplayString(
             ITypeSymbol symbol,
-            CodeAnalysis.NullableFlowState topLevelNullability,
+            CodeAnalysis.NullableFlowState nullableFlowState,
```

Comment:
>nullableFlowState [](start = 43, length = 17)

In the methods above, this parameter is `topLevelNullability`. Should the parameter names match?